test/functorch/test_ops.py

```diff
@@ -1021,23 +1021,21 @@ def fn(inp, *args, **kwargs):
     )
     @ops(op_db + additional_op_db + autograd_function_db, allowed_dtypes=(torch.float,))
     @toleranceOverride({torch.float32: tol(atol=1e-04, rtol=1e-04)})
-    @opsToleranceOverride(
-        "TestOperators",
-        "test_vmapvjpvjp",
-        (
-            tol1("linalg.svd", {torch.float32: tol(atol=1e-03, rtol=5e-04)}),
-            tol1("linalg.lu_factor", {torch.float32: tol(atol=2e-03, rtol=2e-02)}),
-            tol1("svd", {torch.float32: tol(atol=1e-03, rtol=5e-04)}),
-            tol1("matrix_exp", {torch.float32: tol(atol=1e-03, rtol=5e-04)}),
-        ),
-    )
-    @skipOps(
-        "TestOperators",
-        "test_vmapvjpvjp",
-        {
-            xfail("as_strided", "partial_views"),
-        },
-    )
+    @opsToleranceOverride('TestOperators', 'test_vmapvjpvjp', (
```

Comment:
This indentation change is NOT required, revert this change.

Reply:
we have some PRs for different branches to extend tolerance in  `test/functorch/test_ops.py:test_vmapvjpvjp_linalg_householder_product_cuda_float32` 
https://github.com/ROCm/frameworks-internal/issues/7750

Reply:
Comes from commit https://github.com/ROCm/pytorch/commit/9521966dfed08c968507888a26f0c502a576d9a9 to adjust tolerance  for `test/functorch/test_ops.py:test_vmapvjpvjp_linalg_householder_product_cuda_float32`